src/components/Cards/Cards.jsx

```diff
@@ -1,10 +1,23 @@
 import React from "react";
+import SingleCard from "../SingleCard/SingleCard";
 
-function Cards({ category, cardsBySearch }) {
+function Cards({ category, cardsBySearch, singleCard, itemsDetails }) {
```

Comment:
what does cardsBySearch prop do ? and why category is not nested inside itemsDetails
itemsDetails can be renamed to itemDetail since its for 1 item only

Reply:
I replaced category prop by cardBySelectedCategory which returns the card accordıng to the selected category
 and cardsBySearch prop return card by search